### PR TITLE
macOS CI: Updated dependencies and namespaces

### PR DESCRIPTION
**Pull request by PicoCentauri**
Following the discussion of #105 the Qt version in the dependency installation is now hardcoded to version 5.9.2, which is the same as the main obs version for 20.1.0. Furthermore the names are changed from osx to macOS.

### CODE REVIEW

**Comment by Palakis** [comment]
Thanks for this. I hope it will make better macOS builds of obs-websocket.

> the Qt version in the dependency installation is now hardcoded to version 1.9.2

Isn't it 5.9.2? 1.9.2 seems way too old.

**Comment by PicoCentauri** [comment]
> Isn't it 5.9.2? 1.9.2 seems way too old.

Fixed

File: CI/install-dependencies-macos.sh [hunk 1]
```diff
@@ -7,22 +7,26 @@ brew install ffmpeg
 brew install libav
 
 # qtwebsockets deps
-brew install qt5
+# qt latest
+#brew install qt5
+
+# qt 1.9.2
```
Comment (haganbmj) [inline CI/install-dependencies-macos.sh]: super minor nit: `5.9.2`

Comment (PicoCentauri) [reply CI/install-dependencies-macos.sh]: Argh my friend the typo... Should be fixed by now.

**Comment by Palakis** [comment]
Thanks for this. Build seems to work fine.

**Pull request merged**